app/controllers/candidate_interface/gcse/english/grade_controller.rb

```diff
@@ -16,9 +16,7 @@ def update
       @qualification_type = gcse_english_qualification.qualification_type
       @gcse_grade_form = english_gcse_grade_form.assign_values(english_details_params)
 
-      save_successful = multiple_gsces_are_active? && @qualification_type == 'gcse' ? @gcse_grade_form.save_grades : @gcse_grade_form.save_grade
-
-      if save_successful
+      if @gcse_grade_form.save
```

Comment:
I think it's good that the interface to the form is simplified and the logic is now hidden 👍